unlock-app/src/stories/creator/Paywall.stories.js

```diff
@@ -4,38 +4,80 @@ import { storiesOf } from '@storybook/react'
 import Paywall from '../../pages/paywall'
 import createUnlockStore from '../../createUnlockStore'
 
-const lock = {
-  address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
-  keyPrice: '10000000000000000000',
-  expirationDuration: '86400',
-  maxNumberOfKeys: 800,
-  outstandingKeys: 32,
+const myLock = {
+  address: '0xaaaaaaaaa0c4d48d1bdad5dcb26153fc8780f83e',
+  name: 'My Blog',
+  keyPrice: '27000000000000000',
+  expirationDuration: '172800',
+  maxNumberOfKeys: 240,
+  outstandingKeys: 3,
 }
-
-const store = createUnlockStore({
+const lockedState = {
   locks: {
-    '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e': lock,
-    '0xab7c74abc0c4d48d1bdad5dcb26153fc87ffffff': {
-      address: '0xab7c74abc0c4d48d1bdad5dcb26153fc87ffffff',
-      name: 'My Blog',
-      keyPrice: '27000000000000000',
-      expirationDuration: '172800',
-      maxNumberOfKeys: 240,
-      outstandingKeys: 3,
+    '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e': {
+      address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
+      keyPrice: '10000000000000000000',
+      expirationDuration: '86400',
+      maxNumberOfKeys: 800,
+      outstandingKeys: 32,
     },
+    '0xaaaaaaaaa0c4d48d1bdad5dcb26153fc8780f83e': myLock,
   },
   router: {
     location: {
-      pathname: '/demo/' + lock.address,
+      pathname: '/demo/' + myLock.address,
     },
   },
   currency: {
     USD: 195.99,
   },
-})
+}
+const unlockedState = {
+  ...lockedState,
+  account: {
+    address: '0xAaAdEED4c0B861cB36f4cE006a9C90BA2E43fdc2',
+    balance: '989898989898',
+  },
+  keys: {
+    '0xab7c74abc0c4d48d1bdad5dcb26153fc87eeeeee': {
+      lock: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
+      owner: '0xAaAdEED4c0B861cB36f4cE006a9C90BA2E43fdc2',
+      expiration: new Date('December 31, 3000 12:00:00').getTime() / 1000,
+      transaction:
```

Comment:
The transaction should not be an object but the transaction hash.

Reply:
the transaction is a hash, did you mean to say that it should be an object?